lib/cucumber/formatter/backtrace_filter.rb

```diff
@@ -4,13 +4,25 @@
 
 module Cucumber
   module Formatter
-    BACKTRACE_FILTER_PATTERNS = \
-      [/vendor\/rails|lib\/cucumber|bin\/cucumber:|lib\/rspec|gems\/|minitest|test\/unit|.gem\/ruby|lib\/ruby/]
-
-    if(::Cucumber::JRUBY)
-      BACKTRACE_FILTER_PATTERNS << /org\/jruby/
+    @backtrace_filters = %w(
+      /vendor/rails
+      lib/cucumber
+      bin/cucumber:
+      lib/rspec
+      gems/
+      minitest
+      test/unit
+      .gem/ruby
+      lib/ruby/
+      rbenv/.*/bin/bundle
+    )
+
+    if ::Cucumber::JRUBY
```

Comment:
I wonder how helpful this line is - maybe we could just stick this in with the rest of them. WDYT?